src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -79,6 +79,10 @@ error.mechanism:ANR invert-stacktrace=1
 # Note: Newer iOS SDK sends snapshot flag, so this is not strictly necessary.
 error.mechanism:NSError invert-stacktrace=1
 
+# Dart/Flutter stacktraces that are not in-app
+family:javascript stack.abs_path:org-dartlang-sdk* -app -group
```

Comment:
It would be nice to add a test for this. You can, for example, check out https://github.com/getsentry/sentry/pull/24042/files as a reference.

Reply:
everything is inApp=false already, there's no way to detect it automatically because of JS minification, the game-changer here is `-group` which is not testable right now via `test_stacktraces.py`

Reply:
@untitaker should I just merge it or can you point me out on how/where to add a test for this?

Reply:
Add a file to `tests/sentry/grouping/grouping_inputs/` for it to generate test snapshots. As said I can manually approve the snapshots for you if you don't have a working Sentry devenv, after you've done that.

Reply:
@untitaker I pushed a test, can you verify? thanks